getting_started/first_2d_game/03.coding_the_player.rst

```diff
@@ -139,8 +139,27 @@ First, we need to check for input - is the player pressing a key? For this game,
 we have 4 direction inputs to check. Input actions are defined in the Project
 Settings under "Input Map". Here, you can define custom events and assign
 different keys, mouse events, or other inputs to them. For this game, we will
-just use the default events called "ui_right" etc that are assigned to the arrow
-keys on the keyboard.
+map the WASD keys to the four directions.
```

Comment:
I think it's better to stick to arrow keys for this tutorial. That makes the tutorial valid for any keyboard layout, instead of only QWERTY.

Reply:
Alternatively it could use WASD with Physical Keys, but I think that's a nuance better covered in a dedicated input mapping tutorial, so here I'd keep it simple and use arrow keys.

Reply:
Okay, that makes sense.  I can rework it and use the arrow keys rather than WASD.